tools/precommit/check_file_properties.py

```diff
@@ -60,6 +60,7 @@
     r"__T_C_G0",
     r"__YUKAWA",
     r"__cplusplus",
+    r"CP2K_USE_UNIFIED_MEMORY",
```

Comment:
Please don't try to circumvent the checks. It's a new flag that merrits proper documentation in [INSTALL.md](https://github.com/cp2k/cp2k/blob/master/INSTALL.md) and [cp2k_info.F](https://github.com/cp2k/cp2k/blob/master/src/cp2k_info.F).

Reply:
I just realized that `__OFFLOAD_CUDA` and `__OFFLOAD_HIP` are also on the suppression list. I guess this is why you added `CP2K_USE_UNIFIED_MEMORY` as well?

I'm sorry, I should have removed those two when I converted them from internal to external flags.

Reply:
does not matter. I do not want normal users to try this as they have no access to these GPUs. I only have access to them and it is already complicated enough. The PR is to help AMD work.

Reply:
I will remove this flag from the exclusion list.